sql/moz-fx-data-shared-prod/firefox_ios_derived/feature_usage_events_v1/schema.yaml

```diff
@@ -1,238 +1,328 @@
 fields:
-- name: submission_date
+- name: ping_date
```

Comment:
We will also need `submission_date` as for the metrics feature usage table:
https://github.com/mozilla/bigquery-etl/pull/4956/files